test/unit/hd-wallet.ts

```diff
@@ -158,7 +158,7 @@ describe('hd wallet', () => {
         prv: Buffer.from('551d333177df541ad876a60ea71f00447931c0a9da16f227c11ea080d7391b8d', 'hex'),
         pub: Buffer.from('0047150c75db263559a70d5778bf36abbab30fb061ad69f69ece61a72b0cfa4fc0', 'hex')
       }]
-    }]
+    }] as const)
       .forEach(({ seed, tests }, idx) => {
         describe(`vector ${idx + 1}`, () =>
           tests.forEach(test =>
```

Comment:
a bit surprised here how the compiler is picking up the` test.path` type without `as`.